Gemfile.rails_version

```diff
@@ -2,13 +2,16 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV["RAILS_VERSION"] || '4.2.10'
+env_rails_version = '5.1.7'
 
 gem "rails", "~> #{env_rails_version}"
 
 case env_rails_version
 when "master"
   {github: "rails/rails"}
+when /^5.1/
+  gem 'simple_form'
+  gem 'rspec-rails', '~> 3.0'
```

Comment:
updating rspec is a nightmare- not sure how to/if we should handle it in this PR. I was trying to verify that everything worked by using the test suite but it's pretty busted at the moment.

Reply:
Will this not cause a conflict when installing to `rails-server`? Otherwise -- how was this pegged in previous versions of this file?

Reply:
it might- I can remove this line but it will make the specs completely un-runnable.